packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -148,18 +148,21 @@ export function SchemaDrivenAnnotationEditor(
 
   const [showConfirmation, setShowConfirmation] = React.useState(false)
 
-  const { entityMetadata: entityJson, annotations } = useGetJson(
+  const { data } = useGetJson(
     entityId!,
     // Derived annotations will be precomputed and displayed as placeholders in the form
     false,
     {
       // Metadata is being edited, so don't refetch
       staleTime: Infinity,
       enabled: !!entityId,
-      useErrorBoundary: true,
+      throwOnError: true,
     },
   )
 
+  const entityJson = data?.entityMetadata
+  const annotations = data?.annotations
```

Comment:
`useGetJson` splits the Entity's annotations from the system metadata. I changed the hook to put these in the `data` object (a more conventional approach), instead of appending them to the query result.